src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.jsx

```diff
@@ -53,17 +73,150 @@ const ApplyForBenefits = ({ verifiedSavedForms }) => {
+          </div>
+        )}
+        {showChildren && <div className="vads-l-row">{children}</div>}
+        <AdditionalInfo triggerText="What benefits does the VA offer?">
+          <p>Explore VA.gov to learn about the benefits we offer</p>
+          <ul>
+            <li>
+              <a>blah</a>
+            </li>
+            <li>
+              <a>blahhhh</a>
+            </li>
+          </ul>
+        </AdditionalInfo>
+      </div>
+    </>
+  );
+};
+
+const ApplyForBenefits = ({
+  getDD4EDUStatus,
+  getESREnrollmentStatus,
+  hasDD4EDU,
+  hasLoadedAllData,
+  isInESR,
+  isPatient,
+  shouldGetDD4EDUStatus,
+  shouldGetESRStatus,
+  verifiedSavedForms,
+}) => {
+  useEffect(
+    () => {
+      if (shouldGetESRStatus) {
+        getESREnrollmentStatus();
+      }
+    },
+    [shouldGetESRStatus, getESREnrollmentStatus],
+  );
+
+  useEffect(
+    () => {
+      if (shouldGetDD4EDUStatus) {
+        getDD4EDUStatus();
+      }
+    },
+    [shouldGetDD4EDUStatus, getDD4EDUStatus],
+  );
+
+  const showHealthCare = !isPatient && !isInESR;
+  const showEducation = !hasDD4EDU;
+
+  return (
+    <>
+      <h2>Apply for benefits</h2>
+      <ApplicationsInProgress verifiedSavedForms={verifiedSavedForms} />
+      <BenefitsOfInterest showChildren={hasLoadedAllData}>
+        <>
+          {showHealthCare && (
+            <BenefitOfInterest
+              title="Health care"
+              ctaButtonLabel="Apply for health care"
+              ctaUrl="https://va.gov"
+            >
+              <p>yay! content!</p>
+            </BenefitOfInterest>
+          )}
+          <BenefitOfInterest
+            title="File a claim"
+            ctaButtonLabel="File a claim"
+            ctaUrl="https://va.gov"
+          >
+            <p>yay! content about filing a claim with the VA!!!</p>
+            <p>This is two paragraphs just because we can.</p>
+          </BenefitOfInterest>
+          {showEducation && (
+            <BenefitOfInterest
+              title="Education benefits"
+              ctaButtonLabel="Apply for education benefits"
+              ctaUrl="https://va.gov"
+            >
+              <p>Education content!</p>
+            </BenefitOfInterest>
+          )}
+        </>
+      </BenefitsOfInterest>
+    </>
   );
 };
 
 const mapStateToProps = state => {
+  const isPatient = isVAPatient(state);
   const savedForms = selectProfile(state).savedForms || [];
   const verifiedSavedForms = savedForms
     .filter(isSIPEnabledForm)
     .sort(sipFormSorter);
+  const esrEnrollmentStatus = selectESRStatus(state).enrollmentStatus;
+
+  const shouldGetESRStatus = !isPatient;
+  const shouldGetDD4EDUStatus = isMultifactorEnabled(state);
+  const hasLoadedESRData =
+    !shouldGetESRStatus ||
+    hasESRServerError(state) ||
+    esrEnrollmentStatus !== null;
+  const hasLoadedDD4EDUData =
+    !shouldGetDD4EDUStatus || eduDirectDepositInformation(state);
```

Comment:
Is it possible for `eduDirectDepositInformation` to return `null`? Just asking since [I ran into issues with that with the disability rating](https://github.com/department-of-veterans-affairs/vets-website/pull/16021).

Reply:
I'm not sure, but it shouldn't matter. If that selector returns `null` or `undefined`, then `hasLoadedDD4EDUData` will just eval to `false`, which is totally fine in this case :)